package.json

```diff
@@ -26,7 +26,7 @@
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
     "@typescript-eslint/eslint-plugin": "^7.13.1",
-    "@wixc3/define-remix-app": "^4.0.2",
+    "@wixc3/define-remix-app": "^4.2.0",
     "@wixc3/react-board": "^4.0.0",
     "eslint": "^8.56.0",
     "eslint-config-prettier": "^9.1.0",
```

Comment:
I know its not really related but lets make sure we upgrade`"@wixc3/define-remix-app": "^4.0.2",`
     and `"@wixc3/react-board": "^4.0.0",` to their latest. Probably what make this branch have conflicts anyway.

Reply:
will do in another pr later on